src/pages/HomePage.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+// import Categories from '../components/Categories';
+// import Products from '../components/Products';
+// import Newsletter from '../components/Newsletter';
+// import Footer from '../components/Footer';
+// import { desktop, mobile } from '../responsive';
```

Comment:
- [ ] Kindly, remove this piece of old commented code, in order to improve the repo's cleanliness. 👍